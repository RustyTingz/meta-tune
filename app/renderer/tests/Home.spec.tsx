import React from 'react';
import Renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

import Home from '../pages/Home';

test('page Root', () => {
  const component = Renderer.create(
    <MemoryRouter><Home /></MemoryRouter>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
