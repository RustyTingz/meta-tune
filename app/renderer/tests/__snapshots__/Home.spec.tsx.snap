// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page Root 1`] = `
<div
  style={
    Object {
      "marginTop": 100,
      "textAlign": "center",
    }
  }
>
  <h3>
    <a
      href="/importer"
      id="tune-importer"
      onClick={[Function]}
    >
      Tune Importer
    </a>
  </h3>
</div>
`;
